### PR TITLE
Add nuget feed as part of the solution

### PR DESCRIPTION
**Pull request by anddrzejb**
Add nuget feed as part of the solution
Alter documentation for relevant chapters

### CODE REVIEW

File: Chapter-4-applying-tactical-domain-driven-design/README.adoc [hunk 1]
```diff
@@ -88,13 +88,38 @@ That's it! You should now be able to run the application using either one of the
 
 === Building and debugging code in Rider IDE
 
-Before you build or debug code in `Rider` environment, you first have to configure artifactory to allow to download packages for `Common` which is a part of this repository:
+Before you build or debug code in `Rider` or `Visual Studio` IDE, you first have to provide your user name and previously generated PAT for artifactory to download packages for `Common` which is a part of this repository. When you load the solution, your IDE should request the credentials:
```
Comment (kamilbaczek) [inline Chapter-4-applying-tactical-domain-driven-design/README.adoc:91]: `Before you build or debug code in `Rider` or `Visual Studio` IDE, you first have to provide your user name and previously generated PAT for artifactory to download packages for `Common` which is a part of this repository.`

We don't have to pass username and password anymore, we can keep empty when packages are public. 

Please try on your computer additionally if this works, please remove 
"you first have to provide your user name and previously generated PAT for artifactory"

**Review by kamilbaczek — CHANGES_REQUESTED** [review]
Please test and fix approach where you are passing empty username and password

File: Chapter-4-applying-tactical-domain-driven-design/README.adoc (hunk 1 above)
Comment (anddrzejb) [reply Chapter-4-applying-tactical-domain-driven-design/README.adoc:91]: I can't seem to recreate my previous problem, when afer removing the credentials, download was failing. However - without credentials I keep getting the login window. Have a look:
request-login

Comment (kamilbaczek) [reply Chapter-4-applying-tactical-domain-driven-design/README.adoc:91]: Okay, so let's keep like is it.

**Review by kamilbaczek — APPROVED** [review]

**Review by kamilbaczek — APPROVED** [review]
Now it is easier for people to run evolutionary architecure chapters. Thanks!

**Pull request merged**